contracts/switchboard/native/NativeSwitchboardBase.sol

```diff
@@ -297,8 +301,8 @@ abstract contract NativeSwitchboardBase is ISwitchboard, AccessControlExtended {
     }
 
     /**
-     * @notice Allows a watcher to un trip the switchboard by providing a signature and a nonce.
-     * @dev To un trip, the watcher must have the UN_TRIP_ROLE. The signature must be created by signing the concatenation of the following values: "UN_TRIP", the chainSlug, the nonce and false.
+     * @notice Allows a watcher to untrip the switchboard by providing a signature and a nonce.
+     * @dev To untrip, the watcher must have the UNTRIP_ROLE.
```

Comment:
untrip -> un trip
and
UNTRIP_ROLE -> UN_TRIP_ROLE
was done in lot of places in spell checker pr. Lets stick to that change?